### ISSUE TITLE
division by zero not modeled

### ISSUE THREAD

**Issue by crhf** [martinschaef/jar2bpl#6]
It seems that some division by error are missed. For example, `modInt` is modeled as a boogie function, but the mod operation can actually throw when modulus is 0. Do I need to make `modInt` a procedure to model this?

**Comment by martinschaef** [comment]
Yes ... but again, Boogie probably changed since, so I don't know what their semantics is. But, in general, I would model functions if you want to control their behavior.
